helm/values.yaml

```diff
@@ -63,30 +64,56 @@ podAnnotations: {}
 deploymentSelectorMatchLabels:
   app: raw-spans-grouper
 
+statefulSetSelectorMatchLabels:
+  app: raw-spans-grouper
+
+serviceSelectorMatchLabels:
+  app: raw-spans-grouper
+
+volumeClaimTemplates:
+  name: raw-spans-grouper-rocksdb
+  storageClassName: kafka-streams-rocksdb
+  storageRequestSize: 50Gi
+
+jmx:
+  enabled: true
+  port: 7022
+  opts: "-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.port=7022 -Dcom.sun.management.jmxremote.rmi.port=7022 -Djava.rmi.server.hostname=127.0.0.1"
+
+prometheus:
+  jmx:
+    enabled: true
+    port: 7071
+    image:
+      repository: solsson/kafka-prometheus-jmx-exporter@sha256
+      tag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
+      pullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: "0.25"
+        memory: "256Mi"
+
+
+
 ###########
 # Config Maps
 ###########
 rawSpansGrouperConfig:
   name: raw-spans-grouper-config
+  kafka:
+    streams:
+      config:
+        metricsRecordingLevel: INFO
+        numStreamThreads: 4
+        bootstrapServers: "bootstrap:9092"
+        schemaRegistryUrl: "http://schema-registry-service:8081"
+        rocksdbBlockCacheSize: 33554432
```

Comment:
these 3 settings are identically set 2 other times.. wherever is authoritative should comment why the value is this number. ideally it is only set once and not overridden as it increases the amount of things that should be tested.